pom.xml

```diff
@@ -21,11 +21,11 @@
     <parent>
         <artifactId>jahia-modules</artifactId>
         <groupId>org.jahia.modules</groupId>
-        <version>8.0.0.0</version>
+        <version>8.1.8.0</version>
     </parent>
     <artifactId>directive-filter</artifactId>
     <name>Directive Filter</name>
-    <version>2.2.0-SNAPSHOT</version>
+    <version>2.2.0</version>
```

Comment:
This needs to remain 2.2.0-SNAPSHOT, the release process itself will generate the 2.2.0